Wikipedia/Code/WelcomeAnimationExtensions.swift

```diff
@@ -207,6 +207,10 @@ extension UIView{
             return imageView
         })
     }
+
+    func wmf_radiansFromDegrees(degrees: CGFloat) -> CGFloat{
```

Comment:
duplication, make a C function in WMFGeometry so you can call it from Swift. we should get rid of the macro anyway